internal/querynode/query_node.go

```diff
@@ -78,15 +78,9 @@ var Params paramtable.ComponentParam
 //  `indexCoord` is a grpc client of index coordinator.
 //  `stateCode` is current statement of this query node, indicating whether it's healthy.
 type QueryNode struct {
-	queryNodeLoopCtx    context.Context
-	queryNodeLoopCancel context.CancelFunc
-
-	wg sync.WaitGroup
-
+	ctx       context.Context
```

Comment:
why query node still need maintain a context ?

Reply:
This is to keep session, chunk manager, and msgstream as is.